### ISSUE TITLE
order of variable and noise

### ISSUE THREAD

**Issue by jwnys** [maichmueller/scmodels#4]
I'm getting error messages of undefined variables when my assignment string in my assignment map first contains a variable, and only then a noise variable.

For example:
```python
assignment_map = {
   "X1": (
       "N",
       Normal("N", mean=0, std=1)
   ),
   "Y": (
       "X1 + N",
       Normal("N", mean=0, std=1),
   )
}
myscm = SCM(assignment_map)
```

yields the error
```python
NameError: name 'X1' is not defined
```

while the following works:
```python
assignment_map = {
   "X1": (
       "N",
       Normal("N", mean=0, std=1)
   ),
   "Y": (
       "N+X1",
       Normal("N", mean=0, std=1),
   )
}
myscm = SCM(assignment_map)
```

**Comment by maichmueller** [comment]
Thanks for reporting the error @jwnys. This problem should be fixed in version 0.2 soon, which will also allow to specify the SCM with custom callables.
Is this issue urgent for you? If not, I would ask you to simply wait a few more days, until I can get to it.

**Comment by jwnys** [comment]
Hi,
thanks for your reply.
Custom callables would definitely be a good addition (sympy is quite limited, it would be useful to be able to use numpy and scipy!).
The issue above is not hindering me, since I can just put the noise term first. The issue of limiting to sympy, however, is currently putting me to a stop.
